# Solaris-specific configuration defaults.
# Our bash system configuration files are in /etc/bash.
# Not for upstream.
--- doc/bash.1	2011-01-16 12:31:39.000000000 -0800
+++ doc/bash.1	2011-11-29 09:57:25.406978670 -0800
@@ -3937,7 +3937,7 @@
 .B \fIexpr\fP?\fIexpr\fP:\fIexpr\fP
 conditional operator
 .TP
-.B = *= /= %= += \-= <<= >>= &= ^= |=
+.B "= *= /= %= += \-= <<= >>= &= ^= |="
 assignment
 .TP
 .B \fIexpr1\fP , \fIexpr2\fP
@@ -5340,11 +5340,11 @@
 .IP \fB$include\fP
 This directive takes a single filename as an argument and reads commands
 and bindings from that file.  For example, the following directive
-would read \fI/etc/inputrc\fP:
+would read \fI/etc/bash/inputrc\fP:
 .sp 1
 .RS
 .nf
-\fB$include\fP \^ \fI/etc/inputrc\fP
+\fB$include\fP \^ \fI/etc/bash/inputrc\fP
 .fi
 .RE
 .SS Searching
